### DISCUSSION TITLE
about grep

### DISCUSSION THREAD

**Discussion by zhl111** [tadashi-aikawa/obsidian-another-quick-switcher#89] [Q&A] [answered]
I'm sorry I couldn't find help, but I saw that your site was having the same problem I was having, and I wanted to ask you for help.

 `Windows stdio in console mode does not support writing non-UTF-8 byte sequences`

I encountered this while searching for files using `ripgrep-all`, and I couldn't solve it after a long time of torture ......

**Comment by tadashi-aikawa** [comment]
Hi, @zhl111 :)

> Windows stdio in console mode does not support writing non-UTF-8 byte sequences

I have never seen the above error.

> I saw that your site was having the same problem I was having

Would you tell me more details?

**Comment by zhl111** [comment]
Thank you for your reply.
I used the 'rga XXX 'command in Powershell. There are many files that cannot be searched. 'Windows stdio in Console mode does not support writing non-UTF-8 BYTE sequences'.

Strange is, I can not recognize these files in the directory, the execution of 'RGA' search, but normal search out, this problem I have studied for a long time do not know where the problem? And how to solve...
![image](https://user-images.githubusercontent.com/29135984/185733164-dadbf2e4-ef00-4c34-a5dc-1a60de77abe1.png)

**Comment by tadashi-aikawa** [comment]
@zhl111 
Thank you for your description.

So your problem is about ripgrep-all and not about my plugin?

**Comment by zhl111** [comment]
yes,I am sorry...

**Comment by tadashi-aikawa** [comment]
@zhl111 
If your epub file contains non-UTF-8 characters, ripgrep-all doesn't seem support it.

https://github.com/phiresky/ripgrep-all/issues/47

**Comment by zhl111** [comment]
That's very kind of you. Thank you very much. My question seems more complicated, but it still doesn't solve the problem
https://github.com/phiresky/ripgrep-all/issues/47

**Comment by zhl111** [comment]
Is this your website?
![image](https://user-images.githubusercontent.com/29135984/185773803-ec86f73f-f9a7-46f5-b585-38958b66c1e0.png)

**Comment by tadashi-aikawa** [comment]
@zhl111 
Yes. Ah, I have understood why you reached here.

In my case, it occurred when the number of bytes (≈characters) output to the console at one time exceeded a specific value.

If you could share the epub file and steps of reproduction, I might be able to investigate it. (I couldn't reproduce it on my machine.)

**Comment by tadashi-aikawa** [comment, marked as answer]
This issue should convert to discussions, so I'll close it.
